### DISCUSSION TITLE
Is there a way to set higher highlight priority for a specified capture group?

### DISCUSSION THREAD

**Discussion by gh-liu** [nvim-treesitter/nvim-treesitter#4124] [Q&A] [answered]
The semantic tokens will override treesitter's highlight, is there a way to make a specified capture group has higher highlight priority?

For example, the `@text.todo` in `@comment`:
![image](https://user-images.githubusercontent.com/46311996/211247554-1bf4a68a-cd65-405a-b197-aabaabcdb809.png)
![image](https://user-images.githubusercontent.com/46311996/211247611-f2219913-cb15-4303-9c03-a292f2d1bf74.png)

Result of `:Inspect` command(enable semantic tokens):
![image](https://user-images.githubusercontent.com/46311996/211247830-1d1a33f7-ae9d-4e30-8413-1a3ab86639b7.png)

**Comment by clason** [comment, marked as answer]
`:h treesitter-highlight-priority`
